portfolio/src/main/webapp/infiniterun/gamestyle.css

```diff
@@ -0,0 +1,74 @@
+#content {
```

Comment:
Nit: order CSS in alphabetically
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order